Build/Linux/usr/share/doc/atomicgameengine/copyright

```diff
@@ -0,0 +1,1176 @@
+Copyright (c) 2014-2016 THUNDERBEAST GAMES LLC  
+https://github.com/AtomicGameEngine/AtomicGameEngine
```

Comment:
Can this be copied in from the root license file instead of being duplicated, this way it will always be up to date and won't need to be maintained in multiple locations